integration_tests/dbt_project.yml

```
@@ -1,15 +1,15 @@
 name: 'quickbooks_integration_tests'
 
-version: '0.21.1'
+version: '1.0.0'
 
 profile: 'integration_tests'
 config-version: 2
 models:
   +schema: "quickbooks_{{ var('directed_schema','dev') }}"
 
 vars: 
-  quickbooks_source:
-    quickbooks_schema: quickbooks_integration_tests_21
+  quickbooks:
+    quickbooks_schema: quickbooks_integration_tests_22
```

Let's bring this back to original schema name.
```suggestion
    quickbooks_schema: quickbooks_integration_tests
```